src/platforms/common/configuration/filtering.mdx

```diff
@@ -90,7 +90,7 @@ In this example, the fingerprint is forced to a common value if an exception of
 
 </PlatformSection>
 
-<PlatformSection supported={["javascript", "node", "python", "go", "dotnet"]} notSupported={["javascript.cordova"]}>
+<PlatformSection supported={["node", "javascript", "python", "php", "dotnet", "java", "ruby", "go", "react-native", "android", "dotnet"]} notSupported={["javascript.cordova", "flutter"]}>
```

Comment:
which page is this one?

Reply:
The [Filtering](/platforms/flutter/configuration/filtering/) page. It's related to using `tracingSampler` to filter. Since that option isn't available for Flutter, I've turned that content off.